akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -227,6 +228,7 @@ object Attributes {
 
   val none: Attributes = Attributes()
 
+  val verboseLogging: Attributes = Attributes(VerboseLogging)
```

Comment:
I think it's fine to put on Attributes btw; Initially was thinking about ActorAttributes but here is fine 👍